docs/RFC/topbar/2024-05-07.md

```diff
@@ -0,0 +1,50 @@
+# RFC Template
+
+**Feature Name:** topbar
+
+**Start Date:** 2024-05-07
+
+**Author:** Alen Ajam
+
+**Related components/issues:** button, icon-button, link
+
+## Summary
+
+Use the topbar when you want to display an important message, and provides actions for users to address.
+
+## Motivation
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
```

Comment:
Either we put the answer to this question or we remove the section, this is a prompt :laughing:

Reply:
Must've missed it 🤣

Reply:
fixed 👍